available_phone_numbers_test.go

```diff
@@ -35,3 +36,18 @@ func TestSearchAvailablePhoneNumbers(t *testing.T) {
 		t.Errorf("unexpected phone number: %s", res.Numbers[0].PhoneNumber)
 	}
 }
+
+func TestSupportedCountries(t *testing.T) {
+	t.Parallel()
+	client, server := getServer(supportedCountries)
+	defer server.Close()
+
+	res, err := client.AvailableNumbers.SupportedCountries.Get(context.Background(), false)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, country := range res.Countries {
+		fmt.Println(country.Country, country.CountryCode)
```

Comment:
Can we make an assertion here? 2 entries, one of which is Peru?

Reply:
Assertion tests added